src/server/serializer_commons.cc

```diff
@@ -14,11 +14,31 @@ using namespace std;
 
 namespace dfly {
 
+int PackedUIntMeta::Type() const {
```

Comment:
this PR becomes both refactoring and a feature. To simplify the review process I would pull the trivial stuff into a separate commit/PR.

Reply:
or one PR for JournalReader and another for streamer